ting-bill-split.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import "fmt"
+import "flag"
+import "os"
+import "log"
+
+func isBadParam(p *string) bool {
+	return *p == ""
+}
+
+func checkParams(minp *string, msgp *string, megp *string) {
+	badParam := false
+
+	if isBadParam(minp) {
+		fmt.Println("minutes param is bad")
+		badParam = true
+	} else {
+		fmt.Println("minutes:", *minp)
+	}
+
+	if isBadParam(msgp) {
+		fmt.Println("messages param is bad")
+		badParam = true
+	} else {
+		fmt.Println("messages:", *msgp)
+	}
+
+	if isBadParam(megp) {
+		fmt.Println("megabytes param is bad")
+		badParam = true
+	} else {
+		fmt.Println("megabytes:", *megp)
+	}
+
+	if badParam {
+		os.Exit(1)
+	}
+}
+
+func readAndPrint(f *os.File) {
+	data := make([]byte, 100)
+	count, err := f.Read(data)
```

Comment:
Yeah, this was just to get started.  I saw a few of those options and figured I'd lean on suggestions.